modules/dnn/src/layers/pooling_layer.cpp

```diff
@@ -124,6 +124,7 @@ class PoolingLayerImpl CV_FINAL : public PoolingLayer
             CV_Error(Error::StsBadArg, "Cannot determine pooling type");
         setParamsFrom(params);
         ceilMode = params.get<bool>("ceil_mode", true);
+        global_axis = params.get<int>("global_axis", -1);
```

Comment:
I'd like to recommend to introduce `global_pooling_w` and `global_pooling_h` as there are for `kernel` and `stride`.

Reply:
Maybe just a vector like `std::vector<bool> isGlobalPooling`